### PR TITLE
Rgcn forward pass

### PR DESCRIPTION
**Pull request by duncster94**
R-GCN forward pass and `__init__` code now added.

R-GCN includes skip connections from each layer which are summed after the last R-GCN layer. These mention features are then concatenated with the query encoding and passed through a linear dense layer to step feature size down to 1.

Barring future improvements, this should complete the model. Next steps are to add loss function (which involves computing a candidate-masked softmax over the model output) and begin testing.

### CODE REVIEW

**Review by JohnGiorgi — CHANGES_REQUESTED** [review]
`settings.json` got committed by accident, will you drop it?

 ~~Also, do you mind adding the following to the `.gitignore`~~

```
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
```

__I got it__

**Comment by duncster94** [comment]
VS Code saves editor settings in the project directory, last commit should have dropped it.
